x-pack/plugins/enterprise_search/public/applications/shared/kibana_breadcrumbs/generate_breadcrumbs.test.ts

```diff
@@ -181,24 +182,24 @@ describe('appSearchBreadcrumbs', () => {
   describe('links', () => {
     const eventMock = {
       preventDefault: jest.fn(),
-    };
+    } as any;
```

Comment:
One thing I'd really like a sanity check on: I'm using `any` very liberally in our **test** files, where we're manually stubbing/mocking things like event objects (see above example), request bodies, etc. However, Kibana's styleguide/best practices [says to avoid any and to enable an `any` linting rule in our eslint config](https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md#avoid-any-whenever-possible).

What do y'all think between:

1️⃣ We should never use `any` ever including in test files/mocks
2️⃣ We should avoid `any` in source code but allow it in `.test.ts{x}` files
3️⃣ We should hashtag yolo it and use `any` wherever we want

Reply:
I personally vote for `#2` and if we have issues we can't overcome in the production code, let's go for `#3`

Reply:
I agree with Scotty.